### DISCUSSION TITLE
Unable to setup Quorum network using quorum-dev-quickstart on Ubuntu 20.04 .... getting error..???

### DISCUSSION THREAD

**Discussion by ghost** [ConsenSys/quorum#1590] [Q&A]
Hi Team,

I am trying to setup Quorum network using (quorum-dev-quickstart) but getting the below error:
Error response from daemon: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec /usr/local/bin/docker-entrypoint.sh: permission denied: unknown

And i have provided Chmod +X permission to docker-entrypoint.sh but its not working and log file is not getting created for quorum nodes. Node and tessera nodes are running as unhealthy. 

Do you have any recommendation for Ubuntu OS version supported for this Setup?

**Comment by baptiste-b-pegasys** [comment]
Sometimes I need to clean my node modules so I get the latest version of quick dev start. After that the docker compose build with some version of quorum and tessera, it needs to upgrade the image (maybe with docker compose build)

I prefer this repo: https://github.com/ConsenSys/quorum-examples you can run on docker or with locally. It's highly personalizable.
My fork is more up to date.
